### PR TITLE
[Dungeon] Wailing Caverns

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3671;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3670;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3674;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3673;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 201, `maxdmg` = 230, `DamageModifier` = 1.01 WHERE `entry` = 5775;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3653;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3669;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3654;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:35]: fix newline
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3654;

```

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3636;
+UPDATE `creature_template` SET `mindmg` = 43, `maxdmg` = 61, `DamageModifier` = 1.03 WHERE `entry` = 3640;
+UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 59, `DamageModifier` = 1.03 WHERE `entry` = 3840;
+UPDATE `creature_template` SET `mindmg` = 27, `maxdmg` = 38, `DamageModifier` = 1.03 WHERE `entry` = 5053;
+UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 59, `DamageModifier` = 1.03 WHERE `entry` = 5761;
+UPDATE `creature_template` SET `mindmg` = 29, `maxdmg` = 38, `DamageModifier` = 1.03 WHERE `entry` = 5055;
+UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 5756;
+UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 5056;
+UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 5048;
+UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 5755;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:19]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` IN (5756, 5056, 5048, 5755);
```

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ * Update by Knindza | <www.azerothcore.org>
+*/
+
+/* REGULAR */ 
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3637;
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3636;
+UPDATE `creature_template` SET `mindmg` = 43, `maxdmg` = 61, `DamageModifier` = 1.03 WHERE `entry` = 3640;
+UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 59, `DamageModifier` = 1.03 WHERE `entry` = 3840;
+UPDATE `creature_template` SET `mindmg` = 27, `maxdmg` = 38, `DamageModifier` = 1.03 WHERE `entry` = 5053;
+UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 59, `DamageModifier` = 1.03 WHERE `entry` = 5761;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:14]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 59, `DamageModifier` = 1.03 WHERE `entry` IN (3840, 5761);
UPDATE `creature_template` SET `mindmg` = 27, `maxdmg` = 38, `DamageModifier` = 1.03 WHERE `entry` = 5053;
```

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 4]
```diff
@@ -0,0 +1,35 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588336118180516700');
+
+/*
+ * Dungeon: Wailing Caverns
+ * Update by Knindza | <www.azerothcore.org>
+*/
+
+/* REGULAR */ 
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3637;
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3636;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:10]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` IN (3637, 3636);
```

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+/* RARE */														
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 99, `maxdmg` = 127, `DamageModifier` = 1.02 WHERE `entry` = 5912;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3671;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3670;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3674;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3673;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 201, `maxdmg` = 230, `DamageModifier` = 1.01 WHERE `entry` = 5775;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3653;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:33]:
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` IN (3671, 3670, 2673);
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` IN (3674, 3653);
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 201, `maxdmg` = 230, `DamageModifier` = 1.01 WHERE `entry` = 5775;
```

**Comment by Geekena** [comment]
got lost at some points but think i tested all
damage and spells are all ok

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+/* RARE */														
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 99, `maxdmg` = 127, `DamageModifier` = 1.02 WHERE `entry` = 5912;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` IN (3671, 3670, 2673);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` IN (3674, 3653);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 201, `maxdmg` = 230, `DamageModifier` = 1.01 WHERE `entry` = 5775;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3669;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3654;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:27]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` IN (3671, 3670, 2673);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` IN (3674, 3653);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 201, `maxdmg` = 230, `DamageModifier` = 1.01 WHERE `entry` = 5775;
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3669;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3654;
```

File: data/sql/updates/pending_db_world/rev_1588336118180516700.sql [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+UPDATE `creature_template` SET `mindmg` = 65, `maxdmg` = 89, `DamageModifier` = 1.03 WHERE `entry` = 5762;
+
+/* RARE */														
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 99, `maxdmg` = 127, `DamageModifier` = 1.02 WHERE `entry` = 5912;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` IN (3671, 3670, 2673);
+UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` IN (3674, 3653);
+UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 201, `maxdmg` = 230, `DamageModifier` = 1.01 WHERE `entry` = 5775;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3669;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336118180516700.sql:26]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3669;
```

**Pull request merged**